src/globus_cli/services/auth.py

```diff
@@ -8,6 +8,9 @@
 # what qualifies as a valid Identity Name?
 _IDENTITY_NAME_REGEX = r"^[a-zA-Z0-9]+.*@[a-zA-z0-9-]+\..*[a-zA-Z]+$"
 
+# constant for token and login management
+AUTH_RESOURCE_SERVER = "auth.globus.org"
```

Comment:
This placement for this constant is fine, but I would prefer that we define these, and the scopes for these resource servers, as part of the login manager (either in that module or in that class, either is fine). The same applies to the transfer server and scopes.

My reason is that the login manager is responsible for "handling login activity" and I expect it will at some point begin to replace parts of the `globus login` command. At that point, I think it will be convenient and more intuitive for that component to define all of the static scopes and resource servers, as well as the utilities for handling dynamic requirements.

This doesn't need to move now; it's a minor issue and we can address it when we work on more dynamic behaviors.